src/main/java/com/tabnine/general/StaticConfig.java

```diff
@@ -114,14 +123,38 @@ public static String getTabNineBetaVersionUrl() {
         .orElse(getServerUrl() + "/beta_version");
   }
 
-  public static Icon getTabnineIcon(@Nullable ServiceLevel serviceLevel) {
+  public static SubscriptionType getSubscriptionType(@Nullable ServiceLevel serviceLevel) {
```

Comment:
I think that this function should be extracted to a separate class. and add a function `getTabnineIcon` to the `SubscriptionType` enum, such that each enum variant will return its own icon (instead of implementing this with an external switch-case